package_managers/download_pkgs.bzl

```diff
@@ -14,11 +14,26 @@
 
 """Rule for downloading apt packages and tar them in a .tar file."""
 
-load("//package_managers/apt_get:apt_get.bzl", "generate_apt_get")
 load("//package_managers/apt_get:repos.bzl", "generate_additional_repos")
-load("//package_managers:package_manager_provider.bzl", "package_manager_provider")
 load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build")
 
+
+def _generate_download_commands(ctx):
+    return """#!/bin/bash
+set -ex
+# Fetch Index
+apt-get update -y
+# Make partial dir
+mkdir -p {cache}/{archive}/partial
+# Install command
+apt-get install --no-install-recommends -y -q -o Dir::Cache="{cache}" -o Dir::Cache::archives="{archive}" {packages} --download-only
+# Tar command to only include all the *.deb files and ignore other directories placed in the cache dir.
+tar -cpf {output}.tar --directory {cache}/{archive} `cd {cache}/{archive} && ls *.deb`""".format(
+    output=ctx.attr.name,
+    cache='/tmp/install',
```

Comment:
nit: can you just hardcode the ones that we have hardcoded?

Reply:
Done.